src/utils/quick-open.js

```diff
@@ -125,11 +125,13 @@ export function formatShortcutResults(): Array<QuickOpenResult> {
 }
 
 export function formatSources(
-  sources: RelativeSource[],
+  sources: Source[],
   tabs: TabList
 ): Array<QuickOpenResult> {
-  return sources
-    .filter(source => !isPretty(source))
-    .map(source => formatSourcesForList(source, tabs))
-    .filter(({ value }) => value != "");
+  return [
+    ...sources
+      .filter(source => !isPretty(source))
+      .map(source => formatSourcesForList(source, tabs))
+      .filter(({ value }) => value != "")
+  ];
```

Comment:
why can't we do: 

`sources.filter().map().filter()`

if we need to turn this into a JS list we can  do ``sources.filter().map().filter().toJS()`